src/ContentHub.php

```diff
@@ -12,6 +12,10 @@
 
 class ContentHub extends Client
 {
+    // Override VERSION inherited from GuzzleHttp::ClientInterface
+    const VERSION = '0.6.4';
```

Comment:
Just a note to myself: we will need to update this VERSION number every time we make a new release of this PHP library... and that will also include when we release this code 🎯